option optimize_for = LITE_RUNTIME;

message TaskGetRewardRequest
{
 required uint32 id = 1;
}
